www/proxy-sso/src/opnsense/service/conf/actions.d/actions_proxysso.conf

```diff
@@ -0,0 +1,23 @@
+[showkeytab]
+command:( [ ! -f /usr/local/etc/squid/squid.keytab ] && echo "No keytab /usr/local/etc/squid/squid.keytab") || /usr/local/bin/klist -k /usr/local/etc/squid/squid.keytab
+parameters:
+type:script_output
+message:show kerberos keytab
+
+[createkeytab]
+command:/usr/local/opnsense/scripts/OPNsense/ProxySSO/squid-gen-keytab.sh 
+parameters:-n %s -d %s -k %s -e %s -u %s -p %s
+type:script_output
+message:create keytab
+
+[deletekeytab]
+command:( [ ! -f /usr/local/etc/squid/squid.keytab ] && echo "No keytab file" ) || rm /usr/local/etc/squid/squid.keytab
```

Comment:
ideally this should be like a wrapper shell script for keytab operations to avoid bouncing every tweak / change through the service configuration. but this is only a recommendation from my side :)

Reply:
I did not want to produce more essences, than minimum.

Reply:
it's fine for now. if it grows more complex we need to reconsider